develop/tutorials/articles/192-item-selector/01-selecting-entities-using-the-item-selector-api.markdown

```diff
@@ -0,0 +1,356 @@
+# Selecting Entities Using the Item Selector API [](id=selecting-entities-using-the-item-selector-api)
```

Comment:
Hey @jhinkey if you think that "Providing an Item Selector" is better, go ahead with that title. My concern is with the verb "Providing" because to me it sounds more like adding or creating new item selector views, instead of using the item selector to choose entities. But obviously you are the expert here so I trust your decision :)